hw6/assets/style.css

```
@@ -0,0 +1,254 @@
+@font-face {
+    font-family: 'Roboto';
+    src: url('./fonts/Roboto-Regular.ttf') format('truetype');
+    font-weight: 400;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Roboto';
+    src: url('./fonts/Roboto-Bold.ttf') format('truetype');
+    font-weight: 500;
+    font-style: normal;
+}
+
+/* Tmpl main styles */
+
+body {
+    font-family: 'Roboto', sans-serif;
+}
+
+section, header, footer {
+    width: 100%;
+}
+
+header {
+    background: #EEEFF1;
+    position: sticky;
+    top: 0;
+}
+
+footer {
+    background: #34547A;
+}
+
+.navigation, .wrapper {
```

я бы порекомендовал в 2 строчки писать, что б более читабельно было
```suggestion
.navigation,
.wrapper {
```